src/commands/logstream/startStreamingLogs.ts

```diff
@@ -12,15 +12,15 @@ import { localize } from '../../localize';
 import { FunctionAppTreeItem } from '../../tree/FunctionAppTreeItem';
 import { ILogStreamTreeItem } from './ILogStreamTreeItem';
 
-export async function startStreamingLogs(context: vscode.ExtensionContext, tree: AzureTreeDataProvider, node?: IAzureNode<ILogStreamTreeItem>): Promise<void> {
+export async function startStreamingLogs(context: vscode.ExtensionContext, tree: AzureTreeDataProvider, logsOutputChannels: { [channelName: string]: { channelObj: vscode.OutputChannel } }, node?: IAzureNode<ILogStreamTreeItem>): Promise<void> {
```

Comment:
You can just declare `logsOutputChannels` above this function rather than passing it in. That'll keep it scoped to this file and make extension.ts a little less cluttered

Reply:
I tried that but I got some errors where the channel was disabled, In the `extension.ts` scope never happened. We need the extension scope for this to work. Kept it the same for now.

Reply:
Sounds fine